src/screens/Authentication/PhoneNumber/PhoneNumber.tsx

```diff
@@ -26,6 +9,19 @@ function PhoneNumberScreen({
   interface FormValues {
     phoneNumber: string;
   }
+import { useForm, FormProvider, SubmitHandler, SubmitErrorHandler } from 'react-hook-form';
```

Comment:
could you lmk what happened here and can you fix this break in the imports and adding of another PhoneNumberScreen function. 

think this probably happened when pulling some changes from main branch. i'd say just delete the function that's in b/w the imports rn and we can retry the merging